lib/rototiller/task/rototiller_task.rb

```diff
@@ -46,12 +46,14 @@ def self.define_task(*args, &task_block)
       # @option args [String] :name The environment variable
       # @option args [String] :default The default value for the environment variable
       # @option args [String] :message A message describing the use of this variable
+      # @option args [Boolean] :required Indicates whether an error should be raised
+      # if the value is nil or empty string, vs not including the flag.
```

Comment:
No... It really only makes sense in the context of a CommandFlag, but I need it in EnvVar since it is used internally by CommandFlag and contains the logic to determine what the final value for the flag will be.  Since we would never expect a user to specify :required for just an EnvVar, and only expect CommandFlag to use it internally when creating an EnvVar would we want to remove this from the documentation?